task8/src/main/webapp/resources/scripts/postEvents.js

```diff
@@ -0,0 +1,168 @@
+class PostEvents {
+    async setPostEventListener(postElement, postId) {
+        const likePost = postElement.getElementsByClassName('like').item(0);
+        if (likePost) {
+            likePost.addEventListener('click',
+                evt => {
+                    postEvent._updateLike(postId);
+                    evt.stopPropagation();
+                }
+            );
+        }
+
+        const commentPost = postElement.getElementsByClassName('comment').item(0);
+        if (commentPost) {
+            commentPost.addEventListener('click',
+                evt => {
+                    commentEvent.createCommentArea(postId);
+                    evt.stopPropagation();
+                }
+            );
+        }
+
+        const deletePost = postElement.getElementsByClassName('delete').item(0);
+        if (deletePost) {
+            deletePost.addEventListener('click',
+                evt => {
+                    modals.createDeleteModal(postId);
+                    evt.stopPropagation();
+                });
+        }
+
+        const editPost = postElement.getElementsByClassName('edit').item(0);
+        if (editPost) {
+            editPost.addEventListener('click',
+                evt => {
+                    modals.createEditModal(postId);
+                    evt.stopPropagation();
+                })
+        }
+    }
+
+    async setNewPostEventListener(postAreaElement) {
```

Comment:
Perhaps I misunderstood the task, but it says to make a separate pr only for jdbc. and I do not know, to be honest, how you check everything is thrown here just because I thought maybe you will run it.
Anyway, thanks for the answer!